content/docs/gto/user-guide/index.md

```diff
@@ -1,122 +1,220 @@
-[Circle CI](https://circleci.com).
+```yaml
+# deploy/dev.yaml
+object_type: deployment
+type: heroku
+app_name: mlem-dev
+```
 
-The other option is to
-[configure webhooks](https://docs.github.com/en/rest/webhooks) that will send
-HTTP requests to your server upon pushing Git tags to the remote.
+This declaration is read by MLEM in CI and the model promoted to `dev` is
+deployed to https://mlem-dev.herokuapp.com.
 
-Finally, you can configure your server to query your Git provider via something
-like REST API to check if changes happened. As an example, check out
-[Github REST API](https://docs.github.com/en/rest).
+Note, that you need to provide environment variables to deploy to Heroku and
+update the [deployment state](/doc/user-guide/deploying). The location for the
+state should be
+[configured](/doc/user-guide/deploying#setting-up-remote-state-manager) in MLEM
+config file:
+
+```yaml
+# .mlem.yaml
+core:
+  state:
+    uri: s3://bucket/path
+```
 
-### Getting started with CI/CD
+Check out [another example](https://github.com/iterative/example-gto/tree/mlem)
+of MLEM model deployment in the `main` branch of the `example-gto` repo.
 
-To act upon registrations and assignments (Git tags), you can create a simple CI
-workflow. To see an example, check out
-[the workflow in `example-gto` repo](https://github.com/iterative/example-gto/blob/main/.github/workflows/gto-act-on-tags.yml).
-The workflow uses [the GTO GH Action](https://github.com/iterative/gto-action)
-that fetches all Git tags (to correctly interpret the Registry), finds out the
-`version` of the artifact that was registered, the `stage` that was assigned,
-and annotations details such as `path`, `type`, `description`, etc, so you could
-use them in the next steps of the CI.
+</tab>
+</toggle>
 
-### Helpful commands
+## Using DVC to annotate artifacts
 
-If you would like to set up CI/CD, but don't want to use GTO GH Action, check
-out `gto show`, `gto check-ref` and `gto describe` commands.
+Using Git tags to register artifact versions and assign stages is handy, but the
+Git tag itself doesn't contain a path to the artifact files, their type (`model`
+or `dataset`), or any other useful information about them. For simple projects
+(e.g. a single artifact) we can assume the details whenever we consume the
+artifacts (e.g. [in CI/CD](#acting-in-ci-cd)). But for more advanced cases, we
+should codify them in the registry itself.
+
+To keep this metadata,
+[you can use `artifacts:` section in `dvc.yaml` file](https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#artifacts).
+To see an example, check out `dvc.yaml`
+[in the `example-gto` repo](https://github.com/iterative/example-gto).
+
+</admon>
+
+<!-- To download artifact files tracked with DVC, you can use the `dvc get` or
+`dvc import` commands (or simply use `dvc pull` if you `cd` inside the repo).
+
+```cli
+$ dvc get $REPO $ARTIFACT_PATH --rev $REVISION -o $OUTPUT_PATH
+``` -->
```

Comment:
Not deleting this to keep in mind: we need to add `dvc get mymodel#prod` here after it's implemented.